btr-web/btr-layouts/components/bcros/Footer.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <footer
```

Comment:
What was the accessibility error that required you to switch this to div ?

Reply:
"landmark-no-duplicate-contentinfo"

Reply:
I think it would be good that we try to fix this with keeping the footer/header HTML5 tags. If we can. The thing is that they should provide accessibility roles out of the box (header i think is role banner, while footer should be contentinfo)
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/contentinfo_role

Reply:
Yes. It is the header/footer tags in the Fee Summary card that cause the issue. In my recent commit, I have changed them to <div> so we can use <header> and <footer> for the actual header and footer in the app.